File: RecordMusicCunoc/src/main/java/com/adolfosc/modelo/conexion/Respuesta.java
package com.adolfosc.modelo.conexion;

import com.adolfosc.modelo.music.Nota;
import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author hectoradolfo
 */
public class Respuesta {
    
    private String tipo;
    private String nombV;
    private List<CantidadRespuesta> valores;
    private List<Nota> notas;

    public Respuesta() {
        this.valores = new ArrayList<>();
        this.notas = new ArrayList<>();
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getNombV() {
        return nombV;
    }

    public void setNombV(String nombV) {
        this.nombV = nombV;
    }

    public List<CantidadRespuesta> getValores() {
        return valores;
    }

    public void setValores(List<CantidadRespuesta> valores) {
        this.valores = valores;
    }

    public List<Nota> getNotas() {
        return notas;
    }

    public void setNotas(List<Nota> notas) {
        this.notas = notas;
    }
    
    public void insertarValor(CantidadRespuesta valor){
        this.valores.add(valor);
    }
    
    public void insertarNota(Nota nota){
        this.notas.add(nota);
    }
}

File: RecordMusicApp/app/src/main/java/com/adolfosc/modelo/conexion/Respuesta.kt
package com.adolfosc.modelo.conexion

import com.adolfosc.modelo.music.Nota

class Respuesta {

    private var tipo: String? = null
    private var nombV: String? = null
    private var valores: ArrayList<CantidadRespuesta>? = null
    private var notas: ArrayList<Nota>? = null

    constructor() {
        valores = ArrayList()
        notas = ArrayList()
    }

    fun Respuesta() {
        valores = ArrayList()
        notas = ArrayList()
    }

    fun getTipo(): String? {
        return tipo
    }

    fun setTipo(tipo: String?) {
        this.tipo = tipo
    }

    fun getNombV(): String? {
        return nombV
    }

    fun setNombV(nombV: String?) {
        this.nombV = nombV
    }

    fun getValores(): List<CantidadRespuesta>? {
        return valores
    }

    fun setValores(valores: ArrayList<CantidadRespuesta>?) {
        this.valores = valores
    }

    fun getNotas(): List<Nota>? {
        return notas
    }

    fun setNotas(notas: ArrayList<Nota>?) {
        this.notas = notas
    }

    fun insertarValor(valor: CantidadRespuesta) {
        println("valor " + valor)
        valores!!.add(valor)
    }

    fun insertarNota(nota: Nota) {
        notas!!.add(nota)
    }
}